src/App.tsx

```diff
@@ -31,65 +34,130 @@ export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort((good1, good2) => (
+        good1.localeCompare(good2)
+      ));
+
+      break;
+
+    case SortType.LENGTH:
+      visibleGoods.sort((good1, good2) => (
+        good1.length - good2.length
+      ));
+
+      break;
+
+    case SortType.NONE:
```

Comment:
you should return `0`

Reply:
In my case, the function is executed in a switch case, so I don't need to return '0'.